neural_style.py

```diff
@@ -76,21 +84,31 @@ def main():
                 content_image.shape[1] * width)), width)
         content_image = scipy.misc.imresize(content_image, new_shape)
     target_shape = content_image.shape
-    style_image = scipy.misc.imresize(style_image, options.style_scale *
-            target_shape[1] / style_image.shape[1])
+    for i in range(0,num_styles):
+        style_scale = STYLE_SCALE
+        if options.style_scales != None:
+            style_scale = options.style_scales[i]
+        style_images[i] = scipy.misc.imresize(style_images[i], style_scale *
+                target_shape[1] / style_images[i].shape[1])
+
+    style_blend_weights = options.style_blend_weights
```

Comment:
hmm..I was envisioning them passed in as normalized already, but I'll fix this so it auto-normalizes